### PR TITLE
minor tweaks to simplify

### PR DESCRIPTION
**Pull request by winglian**
don't debug the attention mask value since that is almost always exclusively `1` for all cases.
simplify config by not asking users to update the total_num_tokens and sample_packing_eff_est since that computation has been optimized and the optimization to set that ahead of time can cause headaches

### CODE REVIEW

File: src/axolotl/utils/trainer.py [hunk 1]
```diff
@@ -489,6 +489,8 @@ def calculate_total_num_steps(cfg, train_dataset, tokenizer):
             data_loader_len = data_loader.len_w_stats()
             actual_eff = data_loader.efficiency()
             LOG.info(f"data_loader_len: {data_loader_len}")
+            # FIXME: is there a bug here somewhere? the total num steps depends
+            # on the agreed on value for sample_packing_eff_est
```
Comment (NanoCode012) [inline src/axolotl/utils/trainer.py:493]: what does this mean? Do you mean the total num steps should be affected by sample efficiency or some other var?

Comment (winglian) [reply src/axolotl/utils/trainer.py:493]: exactly. because if the various processes have different calculations, we choose the max value to find the smallest steps that they all share. It doesn't seem to be a major issue atm, but should get addressed.
